cmd/support_container_test.go

```diff
@@ -0,0 +1,33 @@
+// go:test !race
```

Comment:
Why does the race detection need to be ignored?

Reply:
This was added by @komish to the support.go. I don't know why though.

Reply:
@komish Is this some leftover from `promptui`? Or is it still needed?

Reply:
Yep, this can be removed. PromptUI leftover.